client/app/scripts/superdesk-authoring/views/authoring-topbar.html

```diff
@@ -87,9 +78,22 @@
                     </li>
                 </ul>
 
-                <ul ng-if="_editable"><li class="divider"></li></ul>
+                <ul ng-if="item.task && item.task.desk && itemActions.mark_item"
```

Comment:
`item.task` is redundant, Angular already gracefully handles cases like `item.<somethingUndefined>.desk`, thusno need to check if `item.task` is a truthy value.